core/src/components/item/test/a11y/e2e.ts

```diff
@@ -6,6 +6,8 @@ test('item: axe', async () => {
     url: '/src/components/item/test/a11y?ionic:_testing=true'
   });
 
-  const results = await new AxePuppeteer(page).analyze();
+  const results = await new AxePuppeteer(page)
```

Comment:
Non-blocking/IDK what the Framework team does here - may help to put a TODO that's tied to a JIRA ticket to re-enable the rules here:
```suggestion
  // TODO(Framework-999): Re-enable these rules
  const results = await new AxePuppeteer(page)
```

Reply:
I did find this bit in Notion: https://www.notion.so/ionic/Code-Reviews-Draft-227f6631f63645cb9170dfd92b14f12a#5612d1764be646ea8dc4933af673db3d

It's still in draft of course, so not something we're "officially" doing yet, but it certainly couldn't hurt here.